locales/en.default.schema.json

```diff
@@ -1801,6 +1870,37 @@
             "label": "Large"
           },
           "info": "For best results, use an image with a 16:9 aspect ratio. [Learn more](https://help.shopify.com/en/manual/shopify-admin/productivity-tools/image-editor#understanding-image-aspect-ratio)"
+        },
+        "desktop_content_alignment": {
+          "options__1": {
+            "label": "Left"
+          },
+          "options__2": {
+            "label": "Center"
+          },
+          "options__3": {
+            "label": "Right"
+          },
+          "label": "Desktop content alignment"
+        },
+        "header": {
+          "content": "Mobile Layout"
```

Comment:
Don't necessarily need to be addressed in this PR, but wanted to bring up some things I noticed. 
cc. @melissaperreault @wiktoriaswiecicka  

I wonder if we want to add vertical positioning when the text box is not used on mobile? This might be useful for merchants that have portrait assets or two images. 

<img width="361" alt="Screen Shot 2021-12-01 at 3 12 42 PM" src="https://user-images.githubusercontent.com/26778637/144306846-72f70bd3-5c47-4605-ac8e-27ea2e10327c.png">

And/or Opacity since merchants can remove text box on mobile but not on desktop. For example, you might keep the text box on desktop and set opacity to zero so that your images are bright/visible. On mobile if you remove the text box, you don't have the option to only add the overlay on mobile.

<img width="1272" alt="Screen Shot 2021-12-01 at 3 10 58 PM" src="https://user-images.githubusercontent.com/26778637/144306619-29c7cca0-5749-4b52-8463-fd679d6d4016.png">


<img width="362" alt="Screen Shot 2021-12-01 at 3 11 34 PM" src="https://user-images.githubusercontent.com/26778637/144306689-10f2dc24-9bc6-49a3-b511-fcc150eb9057.png">

Reply:
@martinamarien, @wiktoriaswiecicka and I actually had this conversation last week 😅 

The reason we decided not to add vertical aligning on mobile for now is because the view for a user would be just of the image and not the content, so we figured itd make the most sense to ensure who's visiting the website to be able to see the content right away 🤔 

But def something we can re-visit 👍

Reply:
That's fair. Makes sense. 

I'd still like to get Wik's input on the opacity on mobile. Because this isn't providing merchants with an accessible option if they choose to remove the textbox on mobile. I understand that it's a specific combination of settings, but think it should still be discussed. We can look into it and if we feel we should add this option, or change the behaviour of the section, we can address it in a different PR.

Reply:
Here is a quick video on how we may need a different opacity setting on mobile:

https://screenshot.click/02-56-nvn3r-vmle8.mp4

Reply:
>  this isn't providing merchants with an accessible option if they choose to remove the textbox on mobile.



Good point. I'm in favour of adding opacity range on mobile. I'll make sure to update the settings on figma and let content review the order and update naming if necessary - so it should be a separate PR. 👍

Reply:
Issue created here: https://github.com/Shopify/dawn/issues/959